Nez.Portable/Assets/Aseprite/AsepriteFrame.cs

```diff
@@ -78,7 +78,14 @@ public Color[] FlattenFrame(bool onlyVisibleLayers = true, bool includeBackgroun
 				//	Note: Will look into adding tilemap cels in a future PR if it is requested enough or if someone
 				//	else wants to add it in.  You can see how I do it in my MonoGame.Aseprite library for reference if
 				//	needed.
-				if (cel is AsepriteImageCel imageCel)
+			CheckCelType:
+				if(cel is AsepriteLinkedCel linkedCel)
+				{
+					cel = linkedCel.Cel;
+					goto CheckCelType;
```

Comment:
Are you sure you forever want your name attached to a goto?!?!

Reply:
Yes